cloudscraper/__init__.py

```diff
@@ -46,7 +46,7 @@
 
 # Add exceptions path
 sys.path.append(os.path.join(os.path.dirname(__file__), 'exceptions'))
-import cloudflare_exceptions  # noqa: E402
+from .exceptions.cloudflare_exceptions import * # noqa: E402
```

Comment:
Don't You think it's better to explicitly name imports?

Reply:
If you're doing relative importing , you might as well remove the sys.path appending for exceptions.

What do we prefer, full name defined objects for the exception... or just the exception name itself?

Reply:
If you're doing relative importing , you might as well  move the sys.path to the exceptions `__init__.py` and remove the noqa E402 from the exceptions import.

So I want to raise (get it ... raise, lol ... jks) this to you guys, as users... What do we prefer, full name defined objects for the exception... or just the exception name itself?

ie 

```python
raise cloudflare_exceptions.someError()
```

or 

```python
raise someError()
```

it gets a little messy when we start putting in other exception classes unrelated to the name with the same error?